app/src/main/java/com/yeonae/chamelezone/view/place/adapter/PlaceDetailPagerAdapter.kt

```diff
@@ -7,11 +7,11 @@ import com.yeonae.chamelezone.view.place.PlaceInfoTabFragment
 import com.yeonae.chamelezone.view.place.PlaceReviewTabFragment
 
 class PlaceDetailPagerAdapter(
-    fm: FragmentManager
+    fm: FragmentManager,  val placeNumber: Int
 ) : FragmentPagerAdapter(fm, BEHAVIOR_RESUME_ONLY_CURRENT_FRAGMENT) {
     override fun getItem(position: Int): Fragment {
         return when(position) {
-            0 -> {PlaceInfoTabFragment()}
+            0 -> {PlaceInfoTabFragment(placeNumber)}
```

Comment:
```suggestion
            0 -> {PlaceInfoTabFragment.newInstance(placeNumber)}
```
fragment 생성자는 parameter가 있으면 lifecycle에서 오류를 초래합니다.
따라서 fragment의 생성자는 parameter가 없어야 합니다.
arguments로 데이터를 주입받아서 사용해야 할 것 같아요.

Reply:
수정했습니당